src/re.rs

```diff
@@ -970,11 +970,11 @@ impl<'t> Index<usize> for Captures<'t> {
 ///
 /// # Panics
 /// If there is no group named by the given value.
-impl<'t> Index<&'t str> for Captures<'t> {
+impl<'t, 'i> Index<&'i str> for Captures<'t> {
```

Comment:
@jld So sorry I missed this comment! You raise a good point. Including those docs in this PR seems fine to me!